api4/team.go

```diff
@@ -340,6 +345,51 @@ func addTeamMember(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(member.ToJson()))
 }
 
+func addTeamMembers(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequireTeamId()
+	if c.Err != nil {
+		return
+	}
+
+	var err *model.AppError
+	members := model.TeamMembersFromJson(r.Body)
+
+	if (len(members) > MAX_ADD_MEMBERS_BATCH || len(members) == 0) {
+		c.SetInvalidParam("")
```

Comment:
Maybe just specify something in the `SetInvalidParam` so the requester knows what field is in correct